### ISSUE TITLE
Add .b2nd as officially supported file extension to the file format specification or use .b2frame otherwise

### ISSUE THREAD

**Issue by kislinsk** [Blosc/c-blosc2#623]
The .b2nd file extension is used in examples and tutorials for Blosc2 files with the b2nd meta layer. However, in the file format specification only .b2frame and .b2f are mentioned as valid file extensions. 

I would expect to either add .b2nd as supported file extension to the specification or to use .b2frame as file extension in all examples and tutorials. At the end of the day that's what it still is according to the header.

**Comment by FrancescAlted** [comment]
Right, there is a new README file describing the current file extensions: https://github.com/Blosc/c-blosc2/blob/main/README_EXTENSION_FILENAMES.rst

Hope this helps.
